### PR TITLE
github actions class.

### PR DESCRIPTION
**Pull request by kikobr82**
# Description

GitHubAction Class

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [X] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/05class/gh-actions/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+<p align="center">
+    <img src="assets/workflows1.png">
+</p>
+
+To get more information about a specific workflow execution, by clicking on it you'll be taken to the workflow logs and you can see all the details and logs for each job/step in your workflow, as the image below:
+<p align="center">
+    <img src="assets/workflows2.png">
+</p>
+
+
```
Comment (denstorti) [inline classes/05class/gh-actions/README.md]: Add link to Marketplace where actions can be found.

File: classes/05class/gh-actions/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      - run: make build
+        working-directory: ./scripts
+      - run: make test
+        working-directory: ./scripts
+      - run: make publish
+        working-directory: ./scripts
+```
+
+Another option would be to run a container as one of the steps of your Job. The code below is an example of that option. 
+
```
[the remark this answers is not in the document]
Comment (kikobr82) [reply classes/05class/gh-actions/README.md]: added in the next commit.

File: classes/05class/gh-actions/README.md (hunk 1 above)
Comment (kikobr82) [reply classes/05class/gh-actions/README.md]: Added in the next commit.

File: classes/05class/gh-actions/README.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+```yaml
+name: Builder
+on: [push]
+
+jobs:
+  my_job:
+    container:
```
[the remark this answers is not in the document]
Comment (kikobr82) [reply classes/05class/gh-actions/README.md]: container is a step:
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer

File: classes/05class/gh-actions/README.md (hunk 1 above)
Comment (denstorti) [reply classes/05class/gh-actions/README.md]: was it added?

Comment (kikobr82) [reply classes/05class/gh-actions/README.md]: it should, but probably missed this commit.. it is added now..:)

**Pull request merged**